homeassistant/components/lupusec/binary_sensor.py

```diff
@@ -14,14 +18,16 @@
 
 SCAN_INTERVAL = timedelta(seconds=2)
 
+_LOGGER = logging.getLogger(__name__)
+
 
 def setup_platform(
     hass: HomeAssistant,
     config: ConfigType,
     add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
-    """Set up a sensor for an Lupusec device."""
+    """Set up a sensor for a Lupusec device."""
     if discovery_info is None:
         return
```

Comment:
We won't be setting up entities via the platform anymore as we are importing the yaml config, so this will never be called -> thus can be removed

Reply:
Removed.